bin/default.nix

```diff
@@ -17,7 +17,7 @@
   playground = writeShellApplication {
     name = "playground";
 
-    runtimeInputs = with pkgs; [
+    runtimeInputs = with pkgs; pkgs.lib.optionals (!stdenv.isDarwin) [
```

Comment:
I don't know Nix well enough, but I wonder if there's a way of making the whole TTY playground Linux-only, rather than putting this safety check here for its dependencies.

Reply:
we could, I've gotten the other parts to run on macOS and some of it helped me catch other issues in `verify-documented-usage`.